src/test/java/uk/gov/justice/hmpps/prison/service/ContactRelationshipIntTest.java

```diff
@@ -5,13 +5,16 @@
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.security.test.context.support.WithMockUser;
 import org.springframework.test.context.ActiveProfiles;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
 import uk.gov.justice.hmpps.prison.api.model.Contact;
 import uk.gov.justice.hmpps.prison.api.model.OffenderRelationship;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @ActiveProfiles("test")
 @SpringBootTest
+@Transactional
```

Comment:
added missing `Transactional` which means that database gets rolled back at end of test and doesn't pollute any others